packages/web-api/src/password-reset/routes.ts

```diff
@@ -0,0 +1,139 @@
+import { prisma } from '@reefguide/db';
+import {
+  PostCreateResetRequestSchema,
+  PostCreateResetResponse,
+  PostUseResetCodeRequestSchema,
+  PostUseResetCodeResponse
+} from '@reefguide/types';
+import express, { Response, Router } from 'express';
+import { processRequest } from 'zod-express-middleware';
+import { EMAIL_SERVICE } from '../config';
+import { BadRequestException, InternalServerError, NotFoundException } from '../exceptions';
+import { PasswordResetService } from './service';
+
+require('express-async-errors');
+export const router: Router = express.Router();
+
+/**
+ * Create a password reset request
+ * POST /password-reset/request
+ */
+router.post(
+  '/request',
+  processRequest({
+    body: PostCreateResetRequestSchema
+  }),
+  async (req, res: Response<PostCreateResetResponse>) => {
+    try {
+      const { email } = req.body;
+      const resetService = new PasswordResetService(prisma);
+
+      console.log('Received password reset request for email:', email);
+      const { code, resetCode } = await resetService.createResetCode({ email });
+      console.log('Generated reset code for email:', email);
+
+      // Send email with reset code
+      try {
+        console.log('Sending password reset email to:', email);
+        // Let this work in the background, don't await
+        EMAIL_SERVICE.sendEmail({
```

Comment:
can wait til another day, but once there are more than a few emails ideally would have them as template files instead of inline in code.